src/sidebar-items.json

```diff
@@ -373,6 +373,17 @@
     "icon": "basket-fill",
     "url": "application-checkout.html"
   },
+  {
+    "name": "Account",
+    "key": "account",
+    "icon": "person-circle",
+    "submenu": [
+      {
+        "name": "Profile",
+        "url": "account-profile.html"
+      }
```

Comment:
Please don't use submenu if there's only one submenu. You can use one menu instead

Reply:
I am working on an account security page which will be added to the Account sub menu, this page contains account settings related to security